### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by maphe**
https://www.pivotaltracker.com/story/show/165257793

### CODE REVIEW

File: adapters/sharethrough/utils.go [hunk 1]
```diff
@@ -13,12 +13,12 @@ import (
...
 func getAdMarkup(strResp openrtb_ext.ExtImpSharethroughResponse, params *hbUriParams) string {
+	var errs []error
+
 	strRespId := fmt.Sprintf("str_response_%s", strResp.BidID)
 	jsonPayload, err := json.Marshal(strResp)
-
 	if err != nil {
-		//handle error
-		fmt.Printf("ERROR: %s\n", err)
+		return ""
```
Comment (madma) [inline adapters/sharethrough/utils.go]: Append to `errs` array also, as below?

File: usersync/usersyncers/syncer.go [hunk 2]
```diff
@@ -2,6 +2,7 @@ package usersyncers
 
 import (
 	"github.com/prebid/prebid-server/adapters/gamoshi"
+	"github.com/prebid/prebid-server/adapters/sharethrough"
```
Comment (madma) [inline usersync/usersyncers/syncer.go]: Only two partners do user syncs?

Comment (madma) [reply usersync/usersyncers/syncer.go]: Never mind, just saw rest of import.

File: adapters/sharethrough/utils.go (hunk 1 above)
Comment (maphe) [reply adapters/sharethrough/utils.go]: Yeah, I tried tried returning the errors but that didn't work out, there's definitely room for improvement here but I wasn't sure if it goes into the 1 point "clean up legacy code" story

I'll suggest another story for reviewing error handling across the adapter

Comment (maphe) [reply adapters/sharethrough/utils.go]: Separate story got created for error handling: https://www.pivotaltracker.com/n/projects/1031566/stories/165745574

**Pull request merged**